internal/controller/controllers/bmh_agent_controller_test.go

```diff
@@ -339,6 +344,49 @@ var _ = Describe("bmac reconcile", func() {
 				Expect(updatedHost.Spec.Online).To(Equal(true))
 				Expect(updatedHost.Spec.AutomatedCleaningMode).To(Equal(bmh_v1alpha1.CleaningModeDisabled))
 			})
+			It("should not reconcile BMH if the image has just been updated", func() {
```

Comment:
```suggestion
			It("should not reconcile BMH if the updated image has not been around longer than the grace period", func() {
```

Or some shorter version of that :D 

We can make it explicit that there is a grace period that we need wait for

Reply:
Done